### PR TITLE
updating web examples to latest

### PR DESCRIPTION
**Pull request by obecny**
Upgrading web examples to latest including the collector docker image

### CODE REVIEW

File: examples/web/docker/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      - ./collector-config.yaml:/conf/collector-config.yaml
+    ports:
+      - "9464:9464"
+      - "4317:4317"
+      - "4318:4318"
+      - "55681:55681"
+    depends_on:
+      - zipkin-all-in-one
+
+  # Zipkin
```
Comment (naseemkullah) [inline examples/web/docker/docker-compose.yaml:18]:
```suggestion
  collector:
    image: otel/opentelemetry-collector:0.38.0
    command: ["--config=/conf/collector-config.yaml"]
    volumes:
      - ./collector-config.yaml:/conf/collector-config.yaml
    ports:
      - "9464:9464"
      - "4317:4317"
      - "4318:4318"
      - "55681:55681"
    depends_on:
      - zipkin-all-in-one

```

Comment (naseemkullah) [inline examples/web/docker/docker-compose.yaml:18]:
```suggestion
```

File: examples/web/docker/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+version: "3"
+services:
+  # Collector
```
Comment (naseemkullah) [inline examples/web/docker/docker-compose.yaml:3]:
```suggestion
```

File: examples/web/docker/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  collector:
+    #    image: otel/opentelemetry-collector:latest
```
Comment (naseemkullah) [inline examples/web/docker/docker-compose.yaml:5]:
```suggestion
```

File: examples/web/package.json [hunk 4]
```diff
@@ -1,10 +1,12 @@
 {
   "name": "web-examples",
   "private": true,
-  "version": "0.23.0",
+  "version": "0.26.0",
   "description": "Example of using web plugins in browser",
   "main": "index.js",
   "scripts": {
+    "docker:start": "cd ./docker && docker-compose down && docker-compose up",
+    "docker:startd": "cd ./docker && docker-compose down && docker-compose up -d",
```
Comment (naseemkullah) [inline examples/web/package.json:9]:
```suggestion
    "docker": "cd ./docker && docker-compose down && docker-compose up",
```

`npm run docker -- -d` might be good enough

**Review by naseemkullah — APPROVED** [review]
lgtm

**Review by vmarchaud — APPROVED** [review]

**Pull request merged**